docs/source/tertiary/whole-genome-regression.rst

```diff
@@ -0,0 +1,352 @@
+===============================
+GlowGR: Whole-Genome Regression
+===============================
+
+.. invisible-code-block: python
+
+    import glow
+    glow.register(spark)
+
+    genotypes_vcf = 'test-data/gwas/genotypes.vcf.gz'
+    covariates_csv = 'test-data/gwas/covariates.csv.gz'
+    continuous_phenotypes_csv = 'test-data/gwas/continuous-phenotypes.csv.gz'
+
+Glow supports Whole Genome Regression (WGR) as GlowGR, a parallelized version of the
+`regenie <https://rgcgithub.github.io/regenie/>`_ method (see the
```

Comment:
Regeneron refer to the method as REGENIE (all caps) in the preprint

Reply:
I'm not sure what the best naming is here; it's described as `regenie` on their software page (https://rgcgithub.github.io/regenie/). @henrydavidge Do you have an opinion?

Reply:
I think I'd go with `regenie`. No strong feelings tho.